src/main/java/com/google/devtools/build/lib/analysis/starlark/Args.java

```diff
@@ -526,8 +526,7 @@ public CommandLineArgsApi useParamsFile(String paramFileArg, Boolean useAlways)
       Starlark.checkMutable(this);
       if (!SingleStringArgFormatter.isValid(paramFileArg)) {
         throw Starlark.errorf(
-            "Invalid value for parameter \"param_file_arg\": Expected string with a single \"%s\"",
-            paramFileArg);
+            "Invalid value for parameter \"param_file_arg\": Expected string with a single \"%%s\"");
```

Comment:
should we still print out `paramFileArg` to ease debugging? after the `a single \"%%s\"` part

Reply:
Good point, changed.

Reply:
@meteorcloudy In case you haven't hit merge yet, could you import the updated version?

Reply:
Sure!

Reply:
Hmm, looks like it somehow broke the bazel_module_test

Reply:
Oh, no, it's my change to the BCR repo, I deleted the `testing` branch, but apparently it's used by the test.

Reply:
I see, it's your base commit is too old, which still uses `https://registry.bazel.build` in the bazel_module_test
https://github.com/bazelbuild/bazel/blob/0529fdb18f42c1381cdefe7534e5b5d86ac1e673/src/test/py/bazel/bzlmod/bazel_module_test.py#L45

But it doesn't matter, the internal presubmit check should pass, since it's always rebased.